events/wc_court_events.txt

```diff
@@ -174,14 +174,45 @@ character_event = {
 	is_triggered_only = yes
 	
 	only_playable = yes
-	min_age = 16
+	min_age = 50
 	prisoner = no
 	
 	trigger = {
+		mercenary = no
+		holy_order = no
+		NOT = { is_offmap_governor = yes }
+		is_theocracy = no
+		OR = {
+			is_republic = no
+			government = merchant_republic_government
+		}
+
+		trigger_if = {
+			limit = {
+				has_game_rule = {
+					name = adoption
+					value = no_player_dynastly
+				}
+				ai = yes
+			}
+			any_dynasty_member = {
+				NOT = { ai = no }
+			}
```

Comment:
Not working, try this:
```
NOT = {
	any_dynasty_member = {
		ai = no
	}
}
```
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/68879825-84b62200-0723-11ea-99e9-74f6fc457fac.png)


</details>

Reply:
fxd